server/controllers/medical/reports/registrations.js

```diff
@@ -39,13 +40,22 @@ function formatFilters(qs) {
     { field : 'debtor_group_uuid', displayName : 'FORM.LABELS.DEBTOR_GROUP' },
     { field : 'patient_group_uuid', displayName : 'PATIENT_GROUP.PATIENT_GROUP' },
     { field : 'user_id', displayName : 'FORM.LABELS.USER' },
+    { field : 'limit', displayName : 'FORM.LABELS.LIMIT' },
+    { field : 'period', displayName : 'FORM.LABELS.PERIODES', isPeriod : true },
```

Comment:
I think we already have a key for this - `TABLE.COLUMNS.PERIOD`.  You should use this one since it will render the same as the one on the client.